admin/workspace-providers.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Workspace Providers
+description: Learn how workspace providers can minimize latency for users.
+state: beta
+---
+
+Coder allows you to add to your deployment multiple Kubernetes clusters. These
+additional clusters are called **providers**. This can be helpful for
+distributed teams, since users can choose the provider closest to them to
+minimize latency.
+
+## Enabling Workspace Providers
+
+To enable the Workspace Providers UI elements:
+
+1. In the Coder UI, click on your avatar.
+2. In the menu that appears, click **Feature Preview**.
+3. Select **Resource Pools UI** and click **Enable**.
+
+Once enabled, this allows end-users to choose the workspace provider and
+namespace where they want their environment deployed.
+
+## Admin UI
+
+Site managers and admins can view the workspace providers configuruation
+page avaiable via **Manage** > **Admin** > **Workspace Providers**.
+
+![Workspace Providers Admin](../assets/workspace-providers-admin.png)
+
+The Admin panel will provide high-level information on all of your workspace
+providers, as well as whether the provider is active on not. The default tag
+indicates the provider that will be automatically selected whenever a user
+launches the Create an Environment modal.
+
+You can expand individual listings to view in-depth information:
+
+![Detailed Workspace Providers Info](../assets/workspace-providers-detail.png)
+
+Workspace Providers are available on a per-organization basis. To make
+individual workspace providers available to organizations:
+
+1. Expand the workspace provider's listing
+2. Click **Edit**
+3. Select the organizations whose users should be allowed to use the workspace
+   provider
+4. Click **Update**.
+
+![Organization Whitelist for Workspace
```

Comment:
I recommend "Allowlist" if possible, though it might be too late to change the terminology for this release, unfortunately, so hopefully we can pick it up next release.

[Slack discussion thread](https://codercom.slack.com/archives/C01CFN1Q2BB/p1611350514005200)

Reply:
I agree (though I think we should keep it as-is until the UI changes as well)